src/jit/rationalize.cpp

```diff
@@ -605,6 +605,9 @@ void Rationalizer::RewriteAssignment(LIR::Use& use)
             }
             JITDUMP("Rewriting GT_ASG(%s(X), Y) to %s(X,Y):\n", GenTree::NodeName(location->gtOper),
                     GenTree::NodeName(storeOper));
+#if NODEBASH_STATS
+            GenTree::RecordOperBashing(storeBlk->OperGet(), storeOper);
+#endif
             storeBlk->gtOper = storeOper;
             storeBlk->gtFlags &= ~GTF_DONT_CSE;
```

Comment:
Yeah, we should at least fix the few occurrences of direct writes to `gtOper` with a follow-up PR.